tsconfig.json

```diff
@@ -22,27 +22,6 @@
     "paths": {
       "@lit-protocol/*": [
         "packages/*/src"
-      ],
```

Comment:
@Ansonhkg @sarahzdev these extra paths were causing build issues for me. We don't need to add any paths as the wildcard `*` is a catchall.

Reply:
these are automatically generated by nx though when you use their command to generate a new package, so if anyone generate a new package it will get reversed back

Reply:
that's understandable but also.... can we agree as a team to not commit that then?

Reply:
sure, we can also create a script to change that to wildcard, so when someone create a new repo it stays the same